alvr/session/src/settings.rs

```diff
@@ -830,10 +830,15 @@ pub struct BodyTrackingFBConfig {
 }
 
 #[derive(SettingsSchema, Serialize, Deserialize, Clone, PartialEq)]
-pub struct BodyTrackingBDConfig {
-    pub high_accuracy: bool,
-    pub prompt_calibration_on_start: bool,
-    pub object_tracking: bool,
+#[schema(gui = "button_group")]
+pub enum BodyTrackingBDConfig {
+    #[schema(strings(display_name = "Body Tracking"))]
+    BodyTracking {
+        high_accuracy: bool,
+        prompt_calibration_on_start: bool,
+    },
+    #[schema(strings(display_name = "Object Tracking"))]
+    ObjectTracking,
```

Comment:
Why this change? the choice to use body or object tracking is done at the client side, so in here you should keep the flat struct to allow for both

Reply:
The only way to find out what mode trackers are using is `xrGetMotionTrackerModeBD` but it is pretty slow as @0JXI0 pointed out.

Reply:
You can separate body tracking but making a separate field and structure maybe. I understand that Pico can do either body tracking or motion tracking but it's not known by the server before connection so both settings paths should be available. similar to this, we expose Nvidia and AMD settings at the same time, because the dashboard doesn't know which GPU is installed/used

Reply:
but it will allow to enable both modes at the same time, right? or should I not care about it?